src/main/java/org/skriptlang/skript/bukkit/itemcomponents/consumable/elements/ExprConsumeEffectApply.java

```diff
@@ -0,0 +1,105 @@
+package org.skriptlang.skript.bukkit.itemcomponents.consumable.elements;
+
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.lang.simplification.SimplifiedLiteral;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.Math2;
+import io.papermc.paper.datacomponent.item.consumable.ConsumeEffect;
+import org.bukkit.event.Event;
+import org.bukkit.potion.PotionEffect;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.consumable.ConsumeEffectExperimentalSyntax;
+import org.skriptlang.skript.registration.SyntaxInfo;
+import org.skriptlang.skript.registration.SyntaxRegistry;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Name("Consume Effect - Apply Effects")
+@Description("""
+	A consume effect that applies the provided potions when the item has been consumed.
+	The probability is the chance the potions get applied when the item has been consumed.
+	Consume effects have to be added to the consumable component of an item.
+	NOTE: Consume Effect elements are experimental. Thus, they are subject to change and may not work as intended.
+	""")
+@Example("""
+	set {_effect} to a consume effect to apply (a new potion effect of strength of tier 3 for 1 hour) with a probability of 100
+	add {_effect} to the consume effects of {_item}
+	""")
+@RequiredPlugins("Minecraft 1.21.3+")
+@Since("INSERT VERSION")
+@SuppressWarnings("UnstableApiUsage")
+public class ExprConsumeEffectApply extends SimpleExpression<ConsumeEffect> implements ConsumeEffectExperimentalSyntax {
+
+	public static void register(SyntaxRegistry registry) {
+		registry.register(
+			SyntaxRegistry.EXPRESSION,
+			SyntaxInfo.Expression.builder(ExprConsumeEffectApply.class, ConsumeEffect.class)
+				.addPatterns("[a] consume effect to apply %potioneffects% with [a] probability of %number%")
```

Comment:
```suggestion
				.addPatterns("[a] consume effect that applies %potioneffects% with [a] probability of %number%")
```
or `[a] chance to apply %potioneffects% with [a] probability of %number%` if you want to get a bit more fluid with the syntax.